shared/templates/template_BASH_regex_permissions

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_all, Red Hat Enterprise Linux 5
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+find FILEPATH -regex '^FILEPATH/FILENAME' -exec chmod FILEMODE {} \;
```

Comment:
I am still not seeing why regex here is needed where `chmod 0600 /boot/System.map.*` makes more sense. Now if you are wanting to find all files that have a specific or unique extension recursively across different file paths and change permissions, then I get using the `find` command with regex.

> In that case I can update the mechanism to use globbing only... and to simplify the chmod directly...

Please do that. This seems a little over-engineered for something that a simple chmod can do.